cve_bin_tool/cli.py

```diff
@@ -279,13 +279,11 @@ def main(argv=None):
         "--epss-percentile",
         action="store",
         help="minimum epss percentile of CVE range between 0 to 100 to report (default: 0)",
-        default=0,
     )
     output_group.add_argument(
         "--epss-probability",
         action="store",
         help="minimum epss probability of CVE range between 0 to 100 to report (default: 0)",
```

Comment:
```suggestion
        help="minimum epss probability of CVE range between 0 to 100 to report",
```

Same as above, no default now so help text needed an update.